extensions/arc/notebooks/arcDeployment/deploy.arc.data.controller.ipynb

```diff
@@ -188,7 +182,7 @@
                 "os.environ[\"AZDATA_PASSWORD\"] = arc_admin_password\n",
                 "if os.name == 'nt':\n",
                 "    print(f'If you don\\'t see output produced by azdata, you can run the following command in a terminal window to check the deployment status:\\n\\t {os.environ[\"AZDATA_NB_VAR_KUBECTL\"]} get pods -n {arc_data_controller_namespace}')\n",
-                "run_command(f'azdata arc dc create --connectivity-mode {arc_data_controller_connectivity_mode} -n {arc_data_controller_name} -ns {arc_data_controller_namespace} -s {arc_subscription} -g {arc_resource_group} -l {arc_data_controller_location} -sc {arc_data_controller_storage_class} --profile-name {arc_profile}')\n",
+                "run_command(f'azdata arc dc create --connectivity-mode Indirect -n {arc_data_controller_name} -ns {arc_data_controller_namespace} -s {arc_subscription} -g {arc_resource_group} -l {arc_data_controller_location} -sc {arc_data_controller_storage_class} --profile-name {arc_profile}')\n",
```

Comment:
Check that the the capitalized `Indirect` is an acceptable value for `--connectivity-mode`.  The help text says the values are "indirect or direct".

Reply:
Yup, it's valid (just tested a deployment with that)